code/modules/mob/abstract/freelook/ai/eye.dm

```diff
@@ -97,3 +97,33 @@
 
 	eyeobj.acceleration = !eyeobj.acceleration
 	to_chat(usr, "Camera acceleration has been toggled [eyeobj.acceleration ? "on" : "off"].")
+
+
+/mob/living/silicon/ai/proc/play_sound()
+	set category = "AI Commands"
+	set name = "Play Music In The Area"
+
+	var/delay = world.time - time_music
+	if(delay < 3000)
+		delay \= 10
+		to_chat(src, "<span class='warning'>You have played music recently, you have to wait [(round(delay/60) > 0) ? ("[round(delay/60)] minutes") : ("[delay] seconds") ]</span>")
+		return
+	var/list/sounds = file2list("sound/serversound_list.txt");
+	sounds += "--CANCEL--"
+	sounds += sounds_cache
+
+	var/melody = input("Select a sound from the server to play", "Server sound list", "--CANCEL--") in sounds
+
+	if(melody == "--CANCEL--")	return
+
+	var/area/A = get_area(eyeobj.loc)
+	if(!A) return

```

Comment:
This is insecure and a potential crash hazard. Navigate AI camera to he outside of the station. Play music. Watch the server chug.

Definitely check if the area is an actual station area! And if it's simulated as well, probably.